bootstrap.sh

```diff
@@ -68,13 +68,19 @@ export PATH=$(pwd)/external/usr/bin:$PATH
 export PYTHONPATH=$(pwd)/external:$(pwd)/external/usr/lib/python$pyver/site-packages:$PYTHONPATH
 
 CMD $python -m pip install --no-cache-dir -q --upgrade pip --target=external/
-CMD $python -m pip install --use-feature=2020-resolver --no-cache-dir -q -r requirements.txt --target=external/ --upgrade
 
-if [ "$MAKEDOCS" == "true" ]; then
+if [ -n "$PYGELF" ]; then
+    cp requirements.txt requirements_tmp.txt
```

Comment:
You should create a real temporary file with  `mktemp`, which you then will delete.